features/compression-dictionary-transport.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2025-03-31
 name: Compression Dictionary Transport
-description: TODO
+description: Compression Dictionary Transport is a mechanism for using designated HTTP responses as an external dictionary for future HTTP responses for compression schemes that support using external dictionaries (e.g., Brotli and Zstandard)
```

@tunetheweb after reading more of the MDN docs, I'd like to propose a description change. We usually try to name the "entry point" of each feature. It seems to me like the most probable one of this feature is the `Use-As-Dictionary`. Is this correct?

```suggestion
description: "The `Use-As-Dictionary` HTTP header allows compression schemes which support dictionaries, such as Brotli or Zstandard, to use HTTP responses as dictionaries for future HTTP responses, making it possible to drastically reduce the compressed size of resources downloaded by the browser."
```

---

No I don’t think that’s a good change.

it’s arguable whether the “entry point” is the `compression-dictionary` link to download the dictionary or the `use-as-dictionary` response on the dictionary. And complicated as for the delta-compression option (I.e. reusing an existing resource as the dictionary) the first step is skipped.

It’s more similar to gzip or Brotli (with some extra negotiation to set it up). I note [the Brotli entry](https://github.com/web-platform-dx/web-features/blob/main/features/brotli.yml) is this:

https://github.com/web-platform-dx/web-features/blob/8c8c996c3c8783ef71627fd4c08aa19f164c53d6/features/brotli.yml#L2

So it’s more a general purpose description and it doesn’t start with `accept-encoding` as the entry point. Though I note it does talk about “content encoding”.

So I’d prefer my original description. Or, alternatively use similar “content encoding” language to the Brotli description.